SIP287 Add C Saturation (#301)
PecanProject/sipnet: docs/parameters.md

| $Q_{10s}$             | soilRespQ10         | Soil respiration Q10                                                                | unitless                                                                            | scalar determining effect of temp on soil respiration          |
| $D_{\text{moisture}}$ | soilRespMoistEffect | scalar determining effect of moisture on soil resp.                                 | unitless                                                                            |                                                                |
| $f_{\text{till}}$     | tillageEff          | Effect of tillage on decomposition that exponentially decays over time              | fraction                                                                            | Documented in model structure; event-level term in `events.in` |

> | $C_{\text{soil,saturation}}$ | soilCSaturation | Maximum amount of carbon that can be stabilized in the soil determined by soil texture        | $\text{g C} \cdot \text{m}^{-2} \text{ ground area}$                                                                                   |                                                                |

Reviewer: current text is ambiguous about who (model or user) is responsible for 'determined by soil texture', and since there is no model representation of this parameter this could be confusing. 

I'd propose something like this, optionally adding "may be estimated from soil texture":

```suggestion
| $C_{\text{soil,saturation}}$ | soilCSaturation | Maximum amount of carbon that can be stabilized in the soil, used when carbon-saturation is enabled.        | $\text{g C} \cdot \text{m}^{-2} \text{ ground area}$                                                                                   |                                                                |
```

Author: Updated

> | $f_{\text{C,soil,saturation}}$ | soilCSaturation | Maximum amount of carbon that can be stabilized in the soil determined by soil texture        | $\text{g C} \cdot \text{m}^{-2} \text{ ground area}$                                                                                   |                                                                |

Reviewer: `f` is reserved for unitless "fraction" parameters.

Something like C_soil,sat, or C^*_soil

Author: Revised to C_soil,saturation
